plugins/linux-capture/CMakeLists.txt

```diff
@@ -47,6 +43,56 @@ target_link_libraries(linux-capture
 	${X11_Xcomposite_LIB}
 	${XCB_LIBRARIES}
 )
+
+option(ENABLE_PIPEWIRE "Enable PipeWire support" ON)
+if(ENABLE_PIPEWIRE)
+	find_package(PipeWire REQUIRED)
+	find_package(Gio REQUIRED)
+
+	set(linux-capture_INCLUDES
+		${linux-capture_INCLUDES}
+		${GIO_INCLUDE_DIRS}
+		${PIPEWIRE_INCLUDE_DIRS}
+	)
+
+	add_definitions(
+		${GIO_DEFINITIONS}
+		${PIPEWIRE_DEFINITIONS}
+	)
+
+	set(linux-capture_SOURCES
+		${linux-capture_SOURCES}
+		pipewire.c
+		pipewire-capture.c
+	)
+	set(linux-capture_HEADERS
+		${linux-capture_HEADERS}
+		pipewire.h
+		pipewire-capture.h
+	)
+	set(linux-capture_LIBRARIES
+		${linux-capture_LIBRARIES}
+		${GIO_LIBRARIES}
+		${PIPEWIRE_LIBRARIES}
+	)
+endif()
+
+configure_file(
```

Comment:
Why is a config file being created and used? Plugins have no public headers and should not be linked against so its hard to find a reason to include this instead of using the definition directly.

Reply:
That's something I still have to update, this is just what I had so far.

Reply:
Done